packages/generator/src/batch/imports.ts

```diff
@@ -19,10 +19,10 @@ export function importBatchDeclarations(
         'GetByKeyRequestBuilder',
         'ODataBatchRequestBuilder',
         'UpdateRequestBuilder',
-        ...(service.functionImports.length
+        ...(service.operationImports.some(op => op.type === 'function')
```

Comment:
[q] Can we also combine `ActionImportRequestBuilder` and `FunctionImportRequestBuilder` into an `OperationRequestBuilder`?